### PR TITLE
Add nomad-driver-containerd to nomad UI docs.

### LINKED ISSUE #5417: [driver plugin] OCI / containerd support
https://containerd.io

> An industry-standard container runtime with an emphasis on simplicity, robustness and portability

Currently it's possible to interact with containerd on nomad only trough the Docker daemon.
It would be great to add the support for interacting directly with containerd without the Docker daemon just like Kubernetes is already doing: https://kubernetes.io/blog/2018/05/24/kubernetes-containerd-integration-goes-ga/
This would mean less memory usage because we wouldn't have to run the Docker daemon to launch the containers just like it's already possible with rkt.

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Fixes #5417

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
ping @DingoEatingFuzz

**Comment by shishir-a412ed** [comment]
@DingoEatingFuzz @jrasell Just checking, if you guys got a chance to review?

File: website/pages/docs/drivers/external/containerd.mdx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+Homepage: https://github.com/Roblox/nomad-driver-containerd
+
+Containerd ([`containerd.io`](https://containerd.io)) is a lightweight container daemon
+for running and managing container lifecycle. Docker daemon also uses containerd.
+
+```hcl
+dockerd (docker daemon) --> containerd --> containerd-shim --> runc
+```
+
+nomad-driver-containerd enables nomad client to launch containers directly using containerd, without docker!
```
Comment (jrasell) [inline website/pages/docs/drivers/external/containerd.mdx:23]:
```suggestion
`nomad-driver-containerd` enables Nomad clients to launch containers directly using containerd, without Docker!
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+Docker daemon is not required on the host system.
```
Comment (jrasell) [inline website/pages/docs/drivers/external/containerd.mdx:24]:
```suggestion
The Docker daemon is therefore not required on the host system.
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        }
+      }
+    }
+  }
+}
+```
+
+The containerd task driver supports the following parameters:
+
+- `image` - (Required) OCI image (docker is also OCI compatible) for your container.
```
Comment (jrasell) [inline website/pages/docs/drivers/external/containerd.mdx:80]:
```suggestion
- `image` - (Required) OCI image (Docker is also OCI compatible) for your container.
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+config {
+  devices = [
+    "/dev/loop0",
+    "/dev/loop1"
+  ]
+}
+```
+- `mounts` - (Optional) A list of mounts to be mounted in the container.
+Volume, bind and tmpfs type mounts are supported. fstab style [`mount options`](https://github.com/containerd/containerd/blob/master/mount/mount_linux.go#L187-L211) are supported.
+   - `type` - (Optional) Supported values are `volume`, `bind` or `tmpfs`. **Default:** volume.
```
Comment (jrasell) [inline website/pages/docs/drivers/external/containerd.mdx:149]:
```suggestion
   - `type` - (Optional) Supported values are `volume`, `bind` or `tmpfs`. **Default:**` volume`.
```

File: website/pages/docs/drivers/external/containerd.mdx (hunk 1 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 2 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 3 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 4 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

**Comment by shishir-a412ed** [comment]
@jrasell Thanks for reviewing it! I have addressed the review comments, PTAL.

**Comment by shishir-a412ed** [comment]
@jrasell I don't see the driver listed in **nomadproject.io** 
docs. 

https://www.nomadproject.io/docs/drivers/external

Anything else needed here?

**Comment by jrasell** [comment]
@shishir-a412ed the website update was released a couple of days ago and the driver documentation is now available.

**Pull request merged**